### PR TITLE
added packages/parany/parany.7.0.0

### PR DESCRIPTION
**Pull request by UnixJunkie**
added core pinning functionality

not that this PR requires this other PR first: https://github.com/ocaml/opam-repository/pull/14602

### CODE REVIEW

File: packages/parany/parany.7.0.0/opam [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+dev-repo: "git+https://github.com/UnixJunkie/parany.git"
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "dune"
+  "base-unix"
+  "ocamlnet"
+  "setcore"
+]
+build: [
+  ["dune" "build" "-p" name]
```
Comment (kit-ty-kate) [inline packages/parany/parany.7.0.0/opam]:
```suggestion
  ["dune" "build" "-p" name "-j" jobs]
```

File: packages/parany/parany.7.0.0/opam [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  ["dune" "runtest" "-p" name] {with-test}
```
Comment (kit-ty-kate) [inline packages/parany/parany.7.0.0/opam]:
```suggestion
  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
```

**Comment by UnixJunkie** [comment]
thanks for the proposed fixes

**Comment by mseri** [comment]
Camelus was complaining for the missing setcore. That is now merged, so I am going to re-run Travis to double check that everything is fine

**Comment by mseri** [comment]
@UnixJunkie can you rebase on master and force push on this PR to force a re-triggering of the CI?

**Comment by kit-ty-kate** [comment]
The tests seem broken:
```
#=== ERROR while compiling parany.7.0.0 =======================================#

# context     2.0.5 | macos/x86_64 | ocaml-base-compiler.4.07.1 | file:///Users/travis/build/ocaml/opam-repository

# path        ~/.opam/ocaml-base-compiler.4.07.1/.opam-switch/build/parany.7.0.0

# command     ~/.opam/opam-init/hooks/sandbox.sh build dune runtest -p parany -j 1

# exit-code   1

# env-file    ~/.opam/log/parany-91430-a00279.env

# output-file ~/.opam/log/parany-91430-a00279.out

### output ###

#         test alias src/runtest (exit 2)

# (cd _build/default/src && ./test.exe 16 800)

# Fatal error: exception File "src/parany.ml", line 176, characters 6-12: Assertion failed
```

**Comment by UnixJunkie** [comment]
I think the test failure is due to the strange CI environment / sandboxing.
The test asks to run on 16 cores (./test.exe 16 ...) but at runtime, the max number of cores detected by Setcore.numcores() was less than that.
It is even OK to comment out the test line in the opam file, because I don't release this library
without testing it.

**Comment by mseri** [comment]
Thanks. I think we can leave it as it is, we can always decide to stop the tests in a separate PR

**Pull request merged**